### PR TITLE
Add MysqlSink

### PR DESCRIPTION
**Pull request by fyqls**

### CODE REVIEW

**Comment by ringtail** [comment]
@fyqls Thanks for contributing !  Could you `make sanitize` first in local.

File: sinks/mysql/mysql.go [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+	sink.Lock()
+	defer sink.Unlock()
+
+	dataPoints := make([]mysql_common.MysqlKubeEventPoint, 0, 10)
+	for _, event := range eventBatch.Events {
+
+		point, err := eventToPoint(event)
+		if err != nil {
+			klog.Warningf("Failed to convert event to point: %v", err)
```
Comment (ringtail) [inline sinks/mysql/mysql.go]: need `continue` to skip the err.

**Comment by ringtail** [comment]
@fyqls Hi, Could you fix the review comment above?

**Comment by fyqls** [comment]
if err != nil {
			klog.Warningf("Failed to convert event to point: %v", err)
			klog.Warningf("Skip this event")
			continue
		}

Add continue to skip this event.

**Pull request merged**